pkg/controller/appconfiguration/keyvalue.go

```diff
@@ -0,0 +1,195 @@
+/*
+Copyright 2021 The Crossplane Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package appconfiguration
+
+import (
+	"context"
+	"strings"
+	"time"
+
+	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/crossplane/crossplane-runtime/pkg/event"
+	"github.com/crossplane/crossplane-runtime/pkg/logging"
+	"github.com/crossplane/crossplane-runtime/pkg/ratelimiter"
+	"github.com/crossplane/crossplane-runtime/pkg/reconciler/managed"
+	"github.com/crossplane/crossplane-runtime/pkg/resource"
+	"github.com/google/go-cmp/cmp"
+	"github.com/pkg/errors"
+	"github.com/stone-payments/appconfig-go-sdk/appconfig/keyvalues"
+	"k8s.io/client-go/util/workqueue"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller"
+
+	"github.com/crossplane/provider-azure/apis/appconfiguration/v1alpha1"
+	azure "github.com/crossplane/provider-azure/pkg/clients"
+	appclient "github.com/crossplane/provider-azure/pkg/clients/appconfiguration"
+)
+
+const (
+	errCheckUpToDate         = "unable to determine if external resource is up to date"
```

Comment:
This constant is duplicated from [client](https://github.com/stone-payments/provider-azure/blob/09c52901689b6240f515453827c069491343fd19/pkg/clients/appconfiguration/keyvalue.go#L29).